apps/system/generic/machine/zynqmp_r5/linker_large_text.ld

```diff
@@ -312,10 +312,10 @@ _SDA2_BASE_ = __sdata2_start + ((__sbss2_end - __sdata2_start) / 2 );
    __undef_stack = .;
 } > psu_r5_tcm_ram_0_S_AXI_BASEADDR
 
-.resource_table 0x3ed20000 : {
+.resource_table _rsc_table : {
 	. = ALIGN(4);
 	*(.resource_table)
-} > psu_ddr_S_AXI_BASEADDR
+}
```

Comment:
Why remove the ' > psu_ddr_S_AXI_BASEADDR'?

Reply:
its possible that the demo is having IPC structures outside of DDR. For example TCM.